drawablepainter/src/main/java/com/google/accompanist/drawablepainter/DrawablePainter.kt

```diff
@@ -58,21 +58,16 @@ private val MAIN_HANDLER by lazy(LazyThreadSafetyMode.NONE) {
 class DrawablePainter(
     val drawable: Drawable
 ) : Painter(), RememberObserver {
-    private var invalidateTick by mutableStateOf(0)
-    private var layoutTick by mutableStateOf(0)
-
-    private var lastIntrinsicSize: Size = Size.Unspecified
+    private var drawInvalidateTick by mutableStateOf(0)
+    private var drawableIntrinsicSize by mutableStateOf(Size.Unspecified)
```

Comment:
maybe to init it with drawable.intrinsicSize, so it has some real size even if DrawablePainter is not remembered and Callback is not set

Reply:
Good call, done.